apps/web/scripts/background/index.ts

```diff
@@ -1,3 +1,26 @@
+import { runtime } from '@walless/core';
+
+(function () {
+	if (runtime.isExtension) {
+		if (chrome && chrome?.runtime?.id) {
+			// Keep Manifest V3 service worker alive
+			chrome.runtime.onInstalled.addListener(() => {
+				chrome.alarms.get('keep-alive', (a) => {
+					if (!a) {
+						chrome.alarms.create('keep-alive', { periodInMinutes: 0.1 });
+					}
+				});
+			});
+		}
+
+		// Add a noop listener to the alarm. Without this, the service worker seems
+		// to be deemed as idle by Chrome and will be killed after 30s.
+		chrome.alarms.onAlarm.addListener(() => {
+			Function.prototype();
+		});
+	}
+})();
+
```

Comment:
don't put the method here, make it a method somewhere and import to this `index.ts` file

Reply:
After discussion, we decided that there was no better place to put this method, so just a small refactoring of the code to make it easier to read